applications/aero_acoustic/co_rotating_vortex_pair/application.h

```diff
@@ -636,6 +636,44 @@ class Application : public ApplicationBase<dim, Number>
 
 namespace AeroAcoustic
 {
+template<int dim>
+class BlendInFunction : public Utilities::SpatialAwareFunction<dim>
+{
+public:
+  BlendInFunction(double const blend_in_start, double const blend_in_end)
+    : Utilities::SpatialAwareFunction<dim>(1, 0.0), start(blend_in_start), end(blend_in_end)
+  {
+  }
+
+  double
+  value(dealii::Point<dim> const &, unsigned int const) const final
+  {
+    return compute_time_factor();
+  }
+
+  bool
+  varies_in_space() const final
+  {
+    return false;
+  }
+
+  double
+  compute_time_factor() const final
+  {
+    double const time = this->get_time();
```

Comment:
As written in another comment, I would not rely here on the dealii design choice of `dealii::Function` with "hidden" time variable. The function `compute_time_factor()` appears to be intrinsically time-dependent, which we should indicate by the interface of the function.

Reply:
I mean this comment here https://github.com/exadg/exadg/pull/644#discussion_r1528639377